### DISCUSSION TITLE
icon to disable/enable easily a camera

### DISCUSSION THREAD

**Discussion by pierreHOH** [blakeblackshear/frigate#5922] [Ideas]
Hi, 
I'm new at frigate (integrated into HA) but I enjoy it a lot compare to old detection only based on simple motion.
I currently use it with several camera. Sometimes, one of them is switched off. 
To prevent to have error messages in logs, It would be useful to have an icon, like "detect", "record" and "snapshot" below the camera picture to enable or disable it easily. 

Except if i am wrong, the only way currently (0.12) is to disable the camera in the config file and restart the frigate application. In this case, there's no longer error messages in logs. Only the fact the camera have been disabled.

From a smartphone it's not very easy to edit the code.

**Comment by NickM-27** [comment]
Well, for there to be an icon it actually has to be supported which is more complicated. There's an existing feature request for this #1911
